metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Creates users from a databag search'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '5.1.0'
+version          '5.1.1'
```

Comment:
per https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD#contribution-dos-and-donts you should not touch the metadata.